services/core/java/com/android/server/policy/PhoneWindowManager.java

```diff
@@ -603,8 +603,10 @@ public void onDrawn() {
     // Screenshot trigger states
     // Time to volume and power must be pressed within this interval of each other.
     private static final long SCREENSHOT_CHORD_DEBOUNCE_DELAY_MILLIS = 150;
+    // Time for volume and power to be pressed to take a screenshot
+    private static final long SCREENSHOT_CHORD_DELAY_TIMER = 0;
     // Increase the chord delay when taking a screenshot from the keyguard
-    private static final float KEYGUARD_SCREENSHOT_CHORD_DELAY_MULTIPLIER = 2.5f;
+    private static final float KEYGUARD_SCREENSHOT_CHORD_DELAY_MULTIPLIER = 1.5f;
```

Comment:
well than, why not make it 0 for both

Reply:
> The reason for the screenshot delay on lockscreen is probably to avoid accidental screenshots when taking the phone out of the pocket etc.

Reply:
Ah I see. For dumpling specifically this is not gonna pose a problem as power button is locked by proximity

Reply:
Gonna leave it as is for other devices tho

Reply:
It's more for the case that the proximity sensor isn't covered anymore (i. e. you just got it out of your pocket) but you haven't turned the screen on yet so at the attempt to press the power button you might accidentally press the volume button as well. This has happened to me before.

Reply:
I understand now. Thank you. Compiling with this now and will see